services/GoogleAdSense.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "Google AdSense",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.google.com/adsense/tc/2018/UnitedKingdom.html",
+      "select": "body"
+    },
+    "Program Policies": {
+      "fetch": "https://support.google.com/adsense/answer/48182",
+      "select": ".article-container",
+      "remove": ".action-button, .zippy"
+    },
+    "Parent Organization Privacy Policy": {
+      "fetch": "https://policies.google.com/privacy?hl=en-GB",
+      "filter": [
+        "removeUTMfromUrls"
+      ],
+      "select": "div[role=article]",
+      "remove": ".KMMDve, .TL2alc, img[src$='.svg']"
+    },
+    "Brand Guidelines": {
+      "fetch": "https://support.google.com/adsense/answer/6370057",
+      "select": ".article-container",
+      "remove": ".no-bullets"
+    },
+    "Quality Guidelines": {
+      "fetch": "https://support.google.com/webmasters/answer/35769",
```

Comment:
Why adding this document to AdSense service? It seems to be global Webmaster Guidelines and independent from AdSense, isn't it?

Reply:
I based myself upon the following sentence of Adsense TOS : "the AdSense Program Policies, which include but are not limited to the Content Policies, the **Webmaster Quality Guidelines**, the Ad Implementation Policies, and the EU User Consent policy", which refers to this [page](https://support.google.com/adsense/answer/1348737?hl=en&ref_topic=1261918), referering itself to the global Webmaster Guidelines :)

Reply:
okay, 👍